smithy-openapi/src/main/java/software/amazon/smithy/openapi/fromsmithy/OpenApiConverter.java

```diff
@@ -302,7 +306,17 @@ private <T extends Trait> OpenApi convertWithEnvironment(ConversionEnvironment<T
         Context<T> context = environment.context;
         OpenApiMapper mapper = environment.mapper;
         OpenApiProtocol<T> openApiProtocol = environment.context.getOpenApiProtocol();
-        OpenApi.Builder openapi = OpenApi.builder().openapi(OpenApiConfig.VERSION).info(createInfo(service));
+        String version = context.getConfig().getVersion();
+
+        // if no version config setting provided, default to v3.0.2
+        if (version == null || version.isEmpty()) {
```

Comment:
We can just move this to  the POJO and get rid of these checks

Reply:
resolved.